partner_credit_limit/model/sale.py

```diff
@@ -30,5 +23,10 @@ def check_limit(self):
                 msg = _('Can not confirm the Sale Order because Partner '
                         'has late payments or has exceeded the credit limit.'
                         '\nPlease cover the late payment or check credit limit'
-                        '\nCreadit Limit : %s') % (so.partner_id.credit_limit)
-                raise exceptions.Warning(('Warning!'), msg)
+                        '\nCredit Limit : %s') % (so.partner_id.credit_limit)
+                raise exceptions.Warning(msg)
+
+    @api.multi
+    def action_confirm(self):
+        self.check_limit()
```

Comment:
Before the call to that function was made by the workflow, but in version 9 there is no sales workflow, so pass the validation call to the function that is responsible for confirming the sales order